pkg/services/ngalert/state/manager.go

```diff
@@ -278,10 +278,14 @@ func (st *Manager) setNextState(ctx context.Context, alertRule *ngModels.AlertRu
 	switch result.State {
 	case eval.Normal:
 		logger.Debug("Setting next state", "handler", "resultNormal")
-		resultNormal(currentState, alertRule, result, logger)
+		// There is no reason for this state as this would be normal behaviour. Normal state will have reason when Error
+		// or NoData states are mapped to OK status by the user.
+		resultNormal(currentState, alertRule, result, logger, "")
```

Comment:
What do you think about adding a constant `NoReason`?

Reply:
It's a good idea. It will be clearer.